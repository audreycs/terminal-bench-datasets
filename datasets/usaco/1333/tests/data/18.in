10
450591848507395078 766
970498285745258150 887
867038654838117493 714
443 698101902806352737
387 449758430496187790
928952692836028738 882
378854113906172456 822
74 220142603208608569
442341090408329812 239
44892686468418987 155998982976513680
