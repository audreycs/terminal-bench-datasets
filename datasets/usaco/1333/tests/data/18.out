588240011106301
1094135609633944
1214339852714501
1575850796402627
1162166487070309
1053234345619137
460893082610948
2974900043359591
1850799541457469
282
